base/libgit2/callbacks.jl

```diff
@@ -24,6 +24,153 @@ function mirror_callback(remote::Ptr{Ptr{Void}}, repo_ptr::Ptr{Void},
     return Cint(0)
 end
 
+function authenticate_ssh(creds::SSHCredentials, cred::Ptr{Ptr{Void}}, username_ptr, schema, host)
```

Comment:
could we give `creds` and `cred` more clearly distinguishable names? this is going to be bug-prone otherwise